src/main/webapp/app/shared/metis/posting-reactions-bar/posting-reactions-bar.component.ts

```diff
@@ -83,6 +89,9 @@ interface ReactionMetaDataMap {
 export class PostingReactionsBarComponent<T extends Posting> implements OnInit, OnChanges {
     protected metisService = inject(MetisService);
     private accountService = inject(AccountService);
+    public conversationService: ConversationService = inject(ConversationService);
+    public modalService: NgbModal = inject(NgbModal);
```

Comment:
I thinks all injects should be `private` also `metisServce`. In case of usage in test, please us `ts-ignore`